### PR TITLE
Build: test out autoddlplugin

### PR DESCRIPTION
**Pull request by samouri**
tests out for speeding up builds in dev: https://github.com/asfktz/autodll-webpack-plugin

seems to double inc build speed (to ~2s) and drop cached builds by 20% (40s --> 32s)

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -211,6 +229,13 @@ if ( calypsoEnv === 'development' ) {
 	webpackConfig.plugins = webpackConfig.plugins.concat( [
 		new webpack.HotModuleReplacementPlugin(),
 		new webpack.LoaderOptionsPlugin( { debug: true } ),
+		new AutoDllPlugin( {
+			debug: true,
+			filename: '[name].js',
```
Comment (blowery) [inline webpack.config.js]: we'll still want a hash in here

File: webpack.config.js [hunk 2]
```diff
@@ -166,35 +167,52 @@ const webpackConfig = {
 	externals: [ 'electron' ]
 };
 
+const vendorLibs = [
+	'classnames',
```
Comment (blowery) [inline webpack.config.js]: how'd you come up with this list? it's pretty different from what we have now.

Comment (samouri) [reply webpack.config.js]: just seeing if i could make the dev speedup greater by putting more in it

**Comment by samouri** [comment]
I should have written a bit more:

the current PR would need extra work to be production ready.
We would only use DDL Plugin for dev and use CommonChunkPlugin for production.

File: webpack.config.js (hunk 1 above)
Comment (samouri) [reply webpack.config.js]: its better to not use hashes during dev (speed/memory reasons).  then use hashes in production builds.

ddl plugin would just be for dev builds

Comment (blowery) [reply webpack.config.js]: how much better? 

Ideally we wouldn't split dev and prod quite that much. Too easy to fall into a trap where dev works and prod bombs.

File: webpack.config.js (hunk 2 above)
Comment (blowery) [reply webpack.config.js]: ah ok. I was hoping you'd worked something up that found all the deps under `node_modules` that `build` depends on and moved them into `vendor` :)

**Pull request closed without merging**